universal-login-sdk/src/integration/http/RelayerApi.ts

```diff
@@ -40,22 +40,22 @@ export class RelayerApi {
     });
   }
 
-  async getPendingAuthorisations(authorisationRequest: RelayerRequest) {
+  getPendingAuthorisations(authorisationRequest: RelayerRequest) {
     const {contractAddress, signature} = authorisationRequest;
     return this.http('GET', `/authorisation/${contractAddress}?signature=${signature}`);
   }
 
-  async cancelConnection(authorisationRequest: RelayerRequest) {
+  cancelConnection(authorisationRequest: RelayerRequest) {
     const {contractAddress} = authorisationRequest;
     return this.http('DELETE', `/authorisation/${contractAddress}`, {authorisationRequest});
   }
 
-  async getConnectedDevices(deviceRequest: RelayerRequest) {
+  getConnectedDevices(deviceRequest: RelayerRequest) {
     const {contractAddress, signature} = deviceRequest;
     return this.http('GET', `/devices/${contractAddress}?signature=${signature}`);
   }
 
-  async deploy(publicKey: string, ensName: string, gasPrice: string, gasToken: string, signature: string, applicationInfo: ApplicationInfo) {
+  deploy(publicKey: string, ensName: string, gasPrice: string, gasToken: string, signature: string, applicationInfo: ApplicationInfo) {
```

Comment:
`Promise<MessageStatus>`

Reply:
Probably Promise<DeploymentStatus>. I remains in that way because it doesn't matter until we will add cast types here from restless